app/src/main/java/com/brandon/forzenbook/repository/ForzenRepositoryImpl.kt

```diff
@@ -87,5 +106,8 @@ class ForzenRepositoryImpl(
         const val DATABASE_ERROR_ACCESS = "Error Accessing Database."
         const val FAILED_ENTITY_CONVERT = "Could Not Convert Response to Entity."
         const val RESPONSE_CODE_ERROR = "Unexpected Response Code."
+
+        const val USER_DUPLICATE_CODE = 409
+        const val USER_ERROR_TEXT = "User Already Exists."
```

Comment:
Created a custom exception for this, `UserAlreadyExistsException`, also cleared up the name of that variable to make it more clear that it is not intended to be shown to the user.

I plan on addressing how its shown to the user in FA-81, where I rework the UI. my reasoning for this is that the error we display to the user might not be the same as "user already exists", since they use an email instead of a username to log in.